brunch/brunch-config.js

```diff
@@ -12,6 +12,13 @@ module.exports = {
       joinTo: {'app.js': /^app\/templates/}
     }
   },
+  npm: {
+    globals: {
+      $: 'jquery',
+      Marionette: 'backbone.marionette',
+      Backbone: 'backbone.marionette/node_modules/backbone'
```

Comment:
I just wanted to simplify app files using global Mn. Most of app files will use $, Mn, Bb and it's became annoying adding each time in top of them smth like `import Marionette from 'backbone.marionette`. That's why I proposed it)
What do you think, is it useless? Should we live it as it is?